test/schema.sql

```diff
@@ -29,15 +29,15 @@ CREATE TABLE spaces (
 );
 
 CREATE TABLE proposals (
-  id VARCHAR(66) NOT NULL,
-  ipfs VARCHAR(64) NOT NULL,
+  id VARCHAR(66) NOT NULL DEFAULT '',
```

Comment:
Don't think we should have default values, specially with `id` we always want it to be random
any reason to add default values?

also not sure we want to update our prod DB with these 🙈  better to change schema of new values only? 😅

Reply:
hmmm correct 😢 werid, i don't understand why we need a default, maybe production has a outdated schema 🥲

Reply:
which way we go?